src/pages/CodeScreen.js

```diff
@@ -37,8 +37,21 @@ class CodeScreen extends Component {
       return;
     }
 
+    if (!msg.data) {
+      this.showError(
+        "Could not get the message from the editor. Please try again",
```

Comment:
nit: "the message" is a little vague here. could be a bit more specific.

Reply:
I agree, but a) this is preexisting, and b) it might as well be an assert: this shouldn't happen in practice.